File: 1523_count_odd_numbers_in_an_interval_range/solution.rs
// LeetCode 1523 - Count Odd Numbers in an Interval Range
// https://leetcode.com/problems/count-odd-numbers-in-an-interval-range/

impl Solution {
    pub fn count_odds(low: i32, high: i32) -> i32 {
        (high + 1) / 2 - low / 2
    }
}

File: 1523_count_odd_numbers_in_an_interval_range/Solution.kt
// LeetCode 1523 - Count Odd Numbers in an Interval Range
// https://leetcode.com/problems/count-odd-numbers-in-an-interval-range/

class Solution {
    fun countOdds(low: Int, high: Int): Int {
        return (high + 1) / 2 - low / 2
    }
}
